index.d.ts

```diff
@@ -684,6 +685,7 @@ declare namespace Eris {
     everyone?: boolean;
     roles?: boolean | string[];
     users?: boolean | string[];
+    replied_user?: boolean;
```

Comment:
the key name here is incorrect, it's actually `repliedUser`

Reply:
i made a fix for that a while ago #1099